gobblin-utility/src/main/java/org/apache/gobblin/data/management/copy/hive/WhitelistBlacklist.java

```diff
@@ -53,23 +53,36 @@
 
   public static final String WHITELIST = "whitelist";
   public static final String BLACKLIST = "blacklist";
+  public static final String IGNORE_CASE = "whitelistBlacklist.ignoreCase";
```

Comment:
Please update the javadoc explaining IGNORE_CASE

Reply:
fixed